scss/detail.scss

```diff
@@ -7,12 +7,27 @@
   align-items: center;
   
   .cover-text-container {
+    position: relative;
     display: flex;
     height: 416px;
     width: 900px;
     background-color: $white;
     box-shadow: 4px 4px 10px 0 $black-rgba;
 
+    .return-button {
+      display: flex;
+      position: absolute;
+      color: $black;
+      font-size: 20px;
+      line-height: 24px;
+      top: -108px;
```

Comment:
I believe that you can use a margin to achieve this positioning without having to use `position: absolute`